susemanager-utils/cluster-providers/caasp/susemanager-cluster-provider-caasp.spec

```diff
@@ -0,0 +1,72 @@
+#
+# spec file for package susemanager-cluster-provider-caasp
```

Comment:
Do we really want to call this `susemanager-` and not `uyuni-`?

Reply:
Ok, I'll change the name to `uyuni-`.